lib/puma/binder.rb

```diff
@@ -383,5 +385,14 @@ def loopback_addresses
     def socket_activation_fd(int)
       int + 3 # 3 is the magic number you add to follow the SA protocol
     end
+
+    def resolve_option(key, conf)
```

Comment:
I see what you're going for here, but I'd prefer to just resolve these values on lines 28-29 without a method.

Reply:
Fair call, thanks for the quick review.